### DISCUSSION TITLE
Cannot disable dragging of the chart inside the container

### DISCUSSION THREAD

**Discussion by SciManJD** [bcakmakoglu/vue-flow#884] [Q&A] [answered]
I'm trying to disable the possibility to move the chart inside the container. According to the documentation (if I understood correct) is to use the option "pan-on-drag" and to set it to false. But the chart still moves.

**Comment by bcakmakoglu** [comment]
Works completely fine for me.
[Here's a sandbox.](https://codesandbox.io/p/sandbox/hungry-johnson-wsnwwu?file=%2Fsrc%2FApp.vue)

**Reply by SciManJD** [reply]
Then I probably don't understand the meaning of the option. I like to block the whole chart of the initial position on the screen, so when I try to move it, it must not move in any direction.

**Reply by bcakmakoglu** [reply]
"Chart"? What chart are you talking about?
Pan on drag disables dragging the viewport as you can see in the sandbox.
If you try to drag somewhere on the background area, nothing will happen.
If you want to disable dragging __nodes__ use the `nodesDraggable` option to globally disable it.

**Reply by SciManJD** [reply]
I used the option nodes-draggable and it works fine. The nodes doesn't move now. But I want to block also the whole diagram (sorry, chart wasn't the right word) inside the window. Now if I catch one of the nodes and try to move it, it is move not separately (because of the nodes-draggable option), but the whole diagram is moved. I want to block it completely to stay always on the position that it is placed.

**Reply by bcakmakoglu** [reply, marked as answer]
Well, use `panOnDrag` together with `nodesDraggable` 😄
